include/slam_toolbox/slam_toolbox_common.hpp

```diff
@@ -65,6 +68,21 @@ class SlamToolbox : public rclcpp::Node
   virtual void configure();
   virtual void loadPoseGraphByParams();
 
+  rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
```

Comment:
`typedef rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn CallbackReturn;`. It saves alot of typing in many locations

Reply:
OK, I'll fix it.

Reply:
Updated to be defined in `toolbox_types.hpp`

https://github.com/SteveMacenski/slam_toolbox/blob/0b8ad824d5e544425bc08bfc629a213b032ca9d0/include/slam_toolbox/toolbox_types.hpp#L126-L130